shiroclient/shiroclient.go

```diff
@@ -381,9 +460,6 @@ const (
 	// MethodShiroPhylum is used to call the ShiroPhylum method which returns
 	// an identifier for the current deployed phylum.
 	MethodShiroPhylum = "ShiroPhylum"
-	// MethodUpgrade is used to call the Upgrade method which upgrades the
-	// substrate .
-	MethodUpgrade = "Upgrade"
```

Comment:
do we not use this anymore?

Reply:
I found it removed in the latest shiroclient, so I ported the deletion into shiroclient-sdk-go